ci/terraform/oidc/api-gateway.tf

```diff
@@ -289,6 +289,10 @@ module "dashboard" {
   use_localstack   = var.use_localstack
 }
 
+data "aws_ssm_parameter" "smoke_test_client_id" {
+  name = "${var.environment}-smoke-test-client-id"
+}
+
```

Comment:
Maybe put this in `ssm.tf`. Are you looking to add the other allowed client ids later?